.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Issue report
+about: Create an issue to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what the issue is.
```

Comment:
Perhaps a more pointed set of questions, like "Explain what commands you ran, what you expected to happen and what actually happened."